src/renderer/src/elements/pocket.ts

```diff
@@ -140,14 +129,10 @@ export class MossPocket extends LitElement {
 
   render() {
     return html`
-      <sl-dialog
-        id="pocket-dialog"
-        style="--width: 800px;"
-        no-header
-        @sl-initial-focus=${(e: { preventDefault: () => void }) => {
-          e.preventDefault();
-          this._searchField.focus();
-        }}
+      <sl-drawer
+        placement="bottom" class="drawer-contained"
+        id="pocket"
+        contained
```

Comment:
The `contained` attribute here seems to disable close on side-click. Do you want to not have it close on side-click intentionally? If so I think I would personally prefer to have the pocket have less height in order to not cover as much tool view port.

Reply:
hmmm.  I thought that you might want to be able to keep the pocket open while doing other things.  If you don't do contained that it's a modal.  But I'm not sure about that.

Reply:
Okay that makes sense. But I have the sense that we should have it cover less vertical space then, what do you think?